staging/src/k8s.io/api/apps/v1beta2/types.go

```diff
@@ -157,6 +157,12 @@ type RollingUpdateStatefulSetStrategy struct {
 	// Default value is 0.
 	// +optional
 	Partition *int32 `json:"partition,omitempty" protobuf:"varint,1,opt,name=partition"`
+	// The maximum number of pods that can be unavailable during the update.
+	// Value can be an absolute number (ex: 5) or a percentage of desired pods (ex: 10%).
```

Comment:
What is 10% of 1? 10% of 2?

Reply:
added a comment, its always rounded up, so it will always be at least 1 and default is also  1 @lavalamp